### ISSUE TITLE
Repository structure for AIXM versions

### ISSUE THREAD

**Issue by manfredbeckmann** [aixm/donlon-outdated-#29]
I have updated the Donlon data set to AIXM 5.1.1 and created a pull request  (#28). Github just allowed me to create the pull request for merging the solitec:aixm-511 branch into the aixm:master branch.

As the community might be interested in having the Donlon data set available in both versions (5.1 and 5.1.1), we should probably have two branches. What do you think?

**Comment by porosnie** [comment]
The AIXM repository in github was created for AIXM 5.1 stuff, we did not think at that time how it would be organised when we have additional AIXM versions. 

Probably it is better to leave it by topics (donlon, wfs-te, etc. - as it is currently), and have versions as sub/branches. For example aixm/donlon/5.1, aixm/donlon/5.1.1, aixm/donlon/5.2 etc.

**Comment by YvesE** [comment]
Just remember that git repository has only one level below the organization "aixm", other sub-levels will folders of the repository. We need also transverse repository like aixm/GML_profile.

Alternative are:
1) aixm/donlon repository with a sub folder per AIXM version.
2) aixm/5.1 and aixm/5.1.1 repositories with sub folders linked to one version like donlon (until now we have only this one)
3) aixm/donlon_5.1 and donlon_5.1.1 repositories

My preference goes to option 1. We should also stated that each donlon version could contain different data as we will not probably report the same modification in all versions.

Any suggestion for the mapping between 2 versions?

**Comment by vog** [comment]
I agree that we should keep aixm/donlon and should not create branches, but sub folders instead (i.e. option 1).

Regarding the mapping, it would be a pity if 5.1 and 5.1.1 went out of sync. We already provide an XSLT script for converting the `gml:id` style (`#fragment` versus `urn:uuid:`). Couldn't we provide an XSLT script for converting Donlon 5.1.1 back to Donlon 5.1 as well (possibly removing all parts of 5.1.1 that are not expressible in 5.1)?

That way, we could continue to improve the 5.1.1 datasets without neglecting the 5.1 datasets.

Additionally, we could add a small script that performs all useful combinations of XSLT scripts (convert to 5.1, convert gml:ids, do both, do nothing), and runs the results through an XML Schema check. That way we could always quickly check that we didn't break anything.
